docs/source/en/_toctree.yml

```diff
@@ -183,6 +183,8 @@
       title: DeepSpeed Integration
     - local: main_classes/feature_extractor
       title: Feature Extractor
+    - local: main/classes/image_processor
```

Comment:
You'll need to write this doc ;-)
Should probably remove the one for feature extractor while we're at it.